src/ImageProcessor.Web/ImageProcessor.Web.csproj

```diff
@@ -32,6 +32,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="ImageProcessor, Version=2.2.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
I'd like to clean this up if possible. Looks like something has changed quite dramatically with references.